src/sql/workbench/services/notebook/browser/models/notebookUtils.ts

```diff
@@ -41,6 +41,10 @@ export function getStandardKernelsForProvider(providerId: string, notebookServic
 		return [];
 	}
 	let standardKernels = notebookService.getStandardKernelsForProvider(providerId);
+	if (!standardKernels || standardKernels.length === 0) {
+		// Fall back to using SQL provider instead
+		standardKernels = notebookService.getStandardKernelsForProvider(SQL_NOTEBOOK_PROVIDER) ?? [];
```

Comment:
Does this have any ramifications re: kernels dropdown if multiple providers have "SQL" as a standard kernel?

Reply:
Or in the model -> notebook provider logic?

Reply:
We would need to have multiple providers for the same file type in order to see that. If you're defining multiple ExecuteProviders for ipynb, then it's fine because you'll be contributing your own standard kernels. You'd only potentially see issues if you're defining multiple SerializationProviders for ipynb, but our code uses a filename-to-provider map to track that type of info, so adding another SerializationProvider for the same file type just overwrites the old one, as far as I know. I'll try testing it and see what happens.

Reply:
Also, a user could contribute their own kernels named "SQL", but that's a separate kind of confusing use case.

Reply:
Small correction, we append new providers to the filename mapping in the notebook service. However, the Kernel dropdown only takes the first kernel for a given display name, and the notebook model only tracks one provider per display name uses a 1-to-1 map. So it seems like there's no issue with a fallback to the SQL kernel.